datashuttle/tui/tabs/create_folders.py

```diff
@@ -153,10 +157,26 @@ def on_clickable_input_clicked(
 
         prefix: Prefix = "sub" if "subject" in input_id else "ses"
 
-        if event.ctrl:
-            self.fill_input_with_template(prefix, input_id)
-        else:
-            self.fill_input_with_next_sub_or_ses_template(prefix, input_id)
+        async def _on_clickable_input_clicked():
```

Comment:
This is nice, it works really nicely, as a slgihtly refactoring, I think 

```
            if event.ctrl:
                self.fill_input_with_template(prefix, input_id)
```
can be moved outside of the async function e.g.
```
    if event.ctrl:
        self.fill_input_with_template(prefix, input_id)
    else:
        async def ....
```